packages/web3-bzz/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "web3-bzz",
-    "version": "1.3.0",
+    "version": "1.3.1-rc.0",
```

Comment:
... @spacesailor24 did you branch off 1.x?

Reply:
Yep, looks like I merged the wrong branch, thank you